tests/vmi_multus_test.go

```diff
@@ -1122,10 +1138,12 @@ var _ = Describe("[Serial]Macvtap", func() {
 	})
 })
 
-func cidrToIP(cidr string) string {
+func cidrToIP(cidr string) (string, error) {
 	ip, _, err := net.ParseCIDR(cidr)
-	Expect(err).ToNot(HaveOccurred(), "Should be able to parse IP and prefix length from CIDR")
-	return ip.String()
+	if err != nil {
+		fmt.Errorf("Failed to parse IP from CIDR: %w", err)
```

Comment:
I believe this should be `return "", fmt.Errorf(...)`

Reply:
Yes, I noticed that forgot to include one staged change, pushed now